media/com_redshop/js/redshop.common.js

```diff
@@ -1178,4 +1178,27 @@ function all_update(u) {
 	document.update_cart.quantity_all.value = q;
 	document.update_cart.task.value = 'update_all';
 	document.update_cart.submit();
+}
+
+function validateInfo() {
```

Comment:
@hungnguyenckc Please also help to change name of function more close its ability. For example: validationUserInfo 
Thanks,
H.A